clientv3/lease.go

```diff
@@ -57,8 +57,10 @@ type Lease interface {
 type lessor struct {
 	c *Client
 
-	mu     sync.Mutex       // guards all fields
-	conn   *grpc.ClientConn // conn in-use
+	mu      sync.Mutex       // guards all fields
+	conn    *grpc.ClientConn // conn in-use
+	initedc chan bool
```

Comment:
I could try in another pr since it requires more code change. just want to get testing working for now. :P